wsl-module/build.sh

```diff
@@ -1,2 +1,3 @@
 #!/bin/sh
+pip freeze > requirements.txt
```

Comment:
I think this should not be here, this is why I deleted it in the last PR.

`requirements.txt` is already created with the necessary packages for running this. If I run this build.sh on my computer, it may get another python environment, creating a wrong `requirements.txt` making the container fail. Also, @0xFugue had problems because this specific line was, for some reasong, importing ubungu libs into `requirements.txt`. So, as requirements is perfectly fine as it is, and it already exists in this repo, this is not needed imo.

Reply:
Oh right. The problem is that someone has to udpate the requirements everytime there are changes in the dependencies of wsl.py One option could be add the pip freeze to the dockerfile.

Reply:
Yes, that is part of the process imo. Like, you are working on your own environment.  If you need new packages, then you update `requirements.txt`, the new file is pushed, but the user should not do `pip freeze` because it may break the process.

Reply:
Ok. Fixed it.